rootfs/usr/local/bin/s3

```diff
@@ -5,7 +5,7 @@ BUCKET=$2
 BUCKET_PREFIX=$3
 MOUNT_POINT=$4
 FSTAB="/etc/fstab"
-FSTYPE="s3fs"
+FSTYPE="/usr/local/bin/s3fs"
```

Comment:
The other fix might be to move `s3fs` to `rootfs/usr/bin`

Reply:
Not against this, it seems like a cleaner solution to just have it in PATH somewhere for future invocations - so s3fs script works as intended every time. I can take a look at this by EOD.

Reply:
crap - looks like we don't. I think we used to. Must have been lost. Let's add it to the `Dockerfile`.

Reply:
@osterman I tried this https://github.com/cloudposse/geodesic/pull/167/commits/4ae1e96665f6930750fc5f81c9af1ca4d4f6431d

Although, it does look like `PATH` at some point does have /usr/local/bin appended to it. The issue seems to be mount -a is not respecting the `PATH` env var. So far the only fix I've been  able to successfully apply has been absolute path in the fstab entry.

Reply:
If not, then let's move `s3fs` to `/usr/bin`